src/gui/shell.h

```diff
@@ -107,6 +107,8 @@ private slots:
 
 	NeovimConnector *m_nvim;
 
+	QList<QUrl> m_urls;
+
```

Comment:
I would prefer this variable to have a more explicit name. But I don't have better alternatives e.g. `m_deferredOpen`